pkg/api/2019-10-27-preview/convertertointernal.go

```diff
@@ -90,8 +90,10 @@ func mergeProperties(oc *OpenShiftManagedCluster, cs *api.OpenShiftManagedCluste
 	if oc.Properties.FQDN != nil {
 		cs.Properties.FQDN = *oc.Properties.FQDN
 	}
-	if oc.Properties.MasterPoolProfile != nil && oc.Properties.MasterPoolProfile.PrivateAPIServer != nil {
-		cs.Properties.PrivateAPIServer = *oc.Properties.MasterPoolProfile.PrivateAPIServer
+	if oc.Properties.MasterPoolProfile != nil && oc.Properties.MasterPoolProfile.APIProperties != nil {
+		if oc.Properties.MasterPoolProfile.APIProperties.PrivateAPIServer != nil {
+			cs.Properties.PrivateAPIServer = *oc.Properties.MasterPoolProfile.APIProperties.PrivateAPIServer
```

Comment:
This internal field stay under the properties structure ? Why not also moving it to the `MasterPoolProfile` ?

Reply:
this is the internal API

Reply:
(the internal API doesn't have a MasterPoolProfile)